person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] Well done @Rafiduzzaman, I appreciate your efforts and patience and I hope you will put in more and more hard effort to be a awesome developer or `engineer`. As mentioned previously by your code reviewer followed by the first code reviewer, you should make the `age` parameter a required parameter, which means if you add something along with age like `age: 28` in the constructor then it  will become `default` parameterized constructor but if you don't pass anything then it will become a normal parameter which always required to pass by user to get an object or exact copy of the class `Person`. I think while making the changes you forgot to remove, in simple words, there is a typo and that is colon "(**:**)" that you passed alongside the age, and because of this small typo I got an error that you can see down below. Kindly fix the issue and you will find that your application will work without any problem.

![image](https://github.com/Rafiduzzaman/Classroom/assets/66476329/373eb0dd-a625-47c2-b121-bf5079f05e0f)

### Suggestion
```suggestion
  def initialize(age , name: 'Unknown', parent_permission: true)
```

Reply:
Hi @skyv26 thanks for the suggestion but I am facing error after removing the colon. here is the video.

https://github.com/Rafiduzzaman/Classroom/assets/103636372/c8b8d5b0-1ff0-4e8d-a724-6a8361ed5937